transforms/helpers/eo-prop/private/abstract.ts

```diff
@@ -82,6 +82,10 @@ export default abstract class AbstractEOProp<
     return !!this.runtimeType;
   }
 
+  get existingDecorators(): Decorator[] | null {
+    return 'decorators' in this._prop ? this._prop.decorators : null;
```

Comment:
Just a suggestion (take it or leave it):




You could change `_prop`:

```diff
- readonly _prop: P;
+ readonly _prop: P & { decorators?: Decorator[] | null };
```

```suggestion
    return this._prop.decorators ?? null;
```